### ISSUE TITLE
Notification mode not saved

### ISSUE THREAD

**Issue by xefil** [nielsfaber/alarmo#283]
### Checklist
- [ X ] I checked for similar existing issues (both open and closed) before posting.
- [ X ] I am running the latest version.

### Expected behavior
Notification script should trigger in any alarm mode

### Actual behavior
Saving the notification script, which has by default "Mode Limit the action to specific arm modes (optional)" all modes selected, if I save, the modes (i.e. Armed Away, Armed Home) are lost. I mean, if I go back to the notification, the Field is empty and in fact the notification gets not triggered. It's possible only to select one (i have only two modes), then the notification is saved and it works for the selected mode.

No errors shown into homeassistant.

### Steps to Reproduce
add notification, don't select any "mode", because by default all are still selected. As soon I save, and go back to the notification in edit mode, the "mode" is empty. I've played with different notifications before, maybe something dirty is remained somewhere? How to check? I've deleted all notifications and recreated, the same

**Comment by xefil** [comment]
In addition, I've the following error for a notification switching to armed:

```

2021-09-03 12:27:45 INFO (MainThread) [custom_components.alarmo.alarm_control_panel] Alarm is armed (armed_away).
2021-09-03 12:27:45 ERROR (MainThread) [homeassistant.util.logging] Exception in async_alarm_state_changed when dispatching 'alarmo_state_updated': ('1629988677', 'arming', 'armed_away')
Traceback (most recent call last):
  File "/config/custom_components/alarmo/automations.py", line 97, in async_alarm_state_changed
    await self.async_execute_automation(automation_id, alarm_entity)
  File "/config/custom_components/alarmo/automations.py", line 168, in async_execute_automation
    for entity_id in alarm_entity.sensors.bypassed_sensors:
AttributeError: 'AlarmoAreaEntity' object has no attribute 'sensors'
```

**Comment by nielsfaber** [comment]
Sorry, I didn't have the time yet to look into this.
The error message, does it occur only right after startup of HA, or at any time you try to arm?

**Comment by nielsfaber** [comment]
I released v1.7.3 which should fix both issues.
Please test it yourself, and close the issue if its indeed fixed, otherwise let me know.

**Comment by xefil** [comment]
Hello @nielsfaber , thanks, seems fixed. No errors and the mods are saved.
Great job
